src/main/java/frc/robot/RobotContainer.java

```diff
@@ -518,6 +535,19 @@ private void configureAutoCommands() {
     Shuffleboard.getTab("MAIN").add(autoChooser.getSendableChooser());
   }
 
+  private void configureIntakeCommands() {
```

Comment:
Let's talk through when we create our own command classes and when we just use the factory methods.